features/fixtures/sampler/android/app/src/main/java/com/sampler/CustomMethodsModule.java

```diff
@@ -0,0 +1,43 @@
+package com.sampler;
+
+import android.util.Log;
+import android.os.Handler;
+import android.os.Looper;
+import android.os.Message;
+import android.os.Bundle;
+import com.facebook.react.bridge.NativeModule;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+import com.bugsnag.android.Bugsnag;
+import com.bugsnag.android.Client;
+import com.bugsnag.android.Configuration;
+
+public class CustomMethodsModule extends ReactContextBaseJavaModule {
```

Comment:
I think longer-term it'd be good to create a separate module for each thing we need to test, i.e. one for ANR, one for session tracking, etc. Otherwise this class could gradually gain more and more responsibilities over time and become large and unwieldy. Maybe we'd want to rename this module something related to ANRs.